reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -513,6 +563,19 @@ default Scheduler newSingle(ThreadFactory threadFactory) {
 		}
 	}
 
+	static ScheduledExecutorService decorateExecutorService(String schedulerType,
+			String description, Supplier<? extends ScheduledExecutorService> supplier) {
+		final Tuple2<String, String> metadata = Tuples.of(schedulerType, description);
+		ScheduledExecutorService result = factory.decorateExecutorService(schedulerType, supplier);
```

Comment:
Before this PR, the provided wrapper executor service would have been the outer most one (since there could be only one) but here it will be the inner most one. I'm wondering if we should keep this behaviour and call `factory.decorateExecutorService` at the end.

Reply:
that makes sense, and easy enough to do 👍